docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -177,21 +187,29 @@ beta::[]
+`alerts`:: Get alerts for {observability}.
 `elasticsearch`:: Call {es} APIs on your behalf.
 `kibana`:: Call {kib} APIs on your behalf.
-`alerts`:: Get alerts for {observability}
-`get_apm_timeseries`:: Display different APM metrics (such as throughput, failure rate, or latency) for any service or all services and any or all of their dependencies. Displayed both as a time series and as a single statistic. Additionally, the function  returns any changes, such as spikes, step and trend changes, or dips. You can also use it to compare data by requesting two different time ranges, or, for example, two different service versions.
-`get_apm_error_document`:: Get a sample error document based on the grouping name. This also includes the stacktrace of the error, which might hint to the cause.
+`summarize`:: Summarize parts of the conversation.
+`visualize_query`:: Visualize charts for ES|QL queries.
+
+Additional functions are available when your cluster has APM data:
+
+[horizontal]
 `get_apm_correlations`:: Get field values that are more prominent in the foreground set than the background set. This can be useful in determining which attributes (such as `error.message`, `service.node.name`, or `transaction.name`) are contributing to, for instance, a higher latency. Another option is a time-based comparison, where you compare before and after a change point.
 `get_apm_downstream_dependencies`:: Get the downstream dependencies (services or uninstrumented backends) for a service. Map the downstream dependency name to a service by returning both `span.destination.service.resource` and `service.name`. Use this to drill down further if needed.
+`get_apm_error_document`:: Get a sample error document based on the grouping name. This also includes the stacktrace of the error, which might hint to the cause.
 `get_apm_service_summary`:: Get a summary of a single service, including the language, service version, deployments, the environments, and the infrastructure that it is running in. For example, the number of pods and a list of their downstream dependencies. It also returns active alerts and anomalies.
 `get_apm_services_list`:: Get the list of monitored services, their health statuses, and alerts.
+`get_apm_timeseries`:: Display different APM metrics (such as throughput, failure rate, or latency) for any service or all services and any or all of their dependencies. Displayed both as a time series and as a single statistic. Additionally, the function  returns any changes, such as spikes, step and trend changes, or dips. You can also use it to compare data by requesting two different time ranges, or, for example, two different service versions.
+
+TIP: Internal functions used by the AI Assistant are not shown here because they are not available for you to suggest.
+You can learn more about these functions by asking the AI Assistant about them. For example, you can ask the AI Assistant,
+"What is the context function?"
```

Comment:
I'm not sure if we should add this. We have four levels of visibility - available all the time, available to the user + Elastic, available to the LLM + Elastic, and Elastic only. E.g. we might move `context` to be an Elastic-only function. I would consider these an implementation detail where it is somewhat of a bug (but inherent to LLMs and not fixable by us) that the LLMs can "tell" the user about these.

Reply:
Removed the tip about internal functions.